autoscaler/kube.py

```diff
@@ -33,6 +33,9 @@ def __init__(self, pod):
         self.status = pod.obj['status']['phase']
         self.uid = metadata['uid']
         self.selectors = pod.obj['spec'].get('nodeSelector', {})
+        # TODO: Remove this, after everyone has migrated off reservations
```

Comment:
I configured our nodes to spin up with `openai.org/reservation-id=default`, so they should still schedule. Once everyone removes this selector from their pods, I'll remove the label from our nodes too